src/pydvl/utils/dataset.py

```diff
@@ -465,6 +467,60 @@ def load_wine_dataset(
     )
 
 
+def load_preprocess_imagenet(
```

Comment:
Since this is support for the notebook, I think it can be moved to the support file that you can find under `/notebooks` in develop.